### PR TITLE
Better understandable text for sending transaction option "Request Replace-By-Fee"

### LINKED ISSUE #11344: Better text for sending transaction option "Request Replace-By-Fee"
### Howto replace a stuck transaction with another with higher fee ?

bitcoind 0.15.0

The marking option "Request Replace-By-Fee" in the sending transaction GUI page should have a better understandable text.

The sender does not "request" anything when creating the transaction. This is just a tag for a later possibility, as I understand as a rather normal user.

I would suggest something like

"Let me increase the fee of this transaction later, if it should not get confirmed in expected time ('Replace-By-Fee' feature)"

### PR DESCRIPTION
**Pull request by ghost**
Fixes #11344

Hope that the changes are technically ok, not sure how the translation string organization works.

### CODE REVIEW

**Comment by MarcoFalke** [comment]
You can drop the second commit. This is automatically done by a script.

File: src/qt/forms/sendcoinsdialog.ui [hunk 1]
```diff
@@ -1108,7 +1108,7 @@
            <item>
             <widget class="QCheckBox" name="optInRBF">
              <property name="text">
-              <string>Request Replace-By-Fee</string>
+              <string>Allow increasing transaction fee later on</string>
```
Comment (MarcoFalke) [inline src/qt/forms/sendcoinsdialog.ui]: Even though within the gui you can only do fee bumping as a result of RBF, in reality RBF allows for more than just increasing the fee. (e.g. additional recipients)

So I think you should at least mention BIP-125 replaceable in parenthesis.

**Comment by MeshCollider** [comment]
Agree with Marco, and it sounds a little weird, I'd prefer something like `Allow transaction to be replaced by one with a higher fee later on (BIP 125)`

**Comment by MeshCollider** [comment]
Please squash commits

**Comment by promag** [comment]
The tooltip already explains the checkbox purpose. *Request Replace-By-Fee* sounds good to me.

**Comment by wodry** [comment]
Can I squash the commits via the Github GUI? I did not find how one could do that. If that is only possible with git commands, unfortunatley that's too complicated for me.

**Comment by MarcoFalke** [comment]
Just noticed the tooltip as well. Agree with @promag that the displayed text does not need clarification. Maybe you could mention BIP 125 in the tooltip?

**Comment by MeshCollider** [comment]
@wodry not sure if its possible with the GUI, but I doubt it. There are instructions on how to do it here: https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#squashing-commits

**Comment by wodry** [comment]
I have seen the tooltip also only when working on this PR, not when doing the transaction as user. The tooltip text is good, but there is a problem with the Tooltip because users and even developers do not see it, as proven here. 
In any case, the button text at the first step should be clearer, in my opinion.
One could also argue the other way round: If the tooltip explains fine, the first text suggestion "Allow increasing transaction fee later on" is really clear to the user, that's what the user effectively get's from this button, and technical details like BIP, and that it is a replacement technique, are to be found in the tooltip text. Clear user feedback from me: "Request Replace-By-Fee" says nothing to a user.

**Comment by wodry** [comment]
I opened an issue for the tooltip problem. #11436

**Comment by flack** [comment]
@promag @MarcoFalke "Request Replace-By-Fee" may be an accurate technical description if you know the underlying implementation, but nothing a casual user can decipher. It's a bit like labelling a hyperlink "perform an HTTP GET request against URI" instead of "go to this page". IMHO the initially visible text should give an indication of what it does in a way the user can relate to (and what it does is it allows you to use "bump fee" later on, right?). The technical details (like BIP number and technical acronyms) should go to the tooltip, since it's a "would you like to know more?" type of thing

**Comment by laanwj** [comment]
Screenshot please?

**Comment by wodry** [comment]
I tried to squash the six commits according to

https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#squashing-commits

git rebase -i HEAD~6

did nothing. Now it seems, that I messed it up. Sorry. If anyone could give me some commands for git console to fix it, I would be thankful.

**Comment by MarcoFalke** [comment]
```sh
git reset --soft 90926db2381d87c68858659873230a3811ebdce5 && git commit

**Comment by wodry** [comment]
Here is the screenshot of current 0.15.0.1 GUI send transaction page. Have no screenshot of the window after this change, have no capacitiy to build and run it.
bildschirmfoto

**Comment by jonasschnelli** [comment]
I prefer to wording in master, though I agree ">Request< Replace-By-Fee" is maybe not ideal from the users perspective.

File: src/qt/forms/sendcoinsdialog.ui (hunk 1 above)
Comment (laanwj) [reply src/qt/forms/sendcoinsdialog.ui]: Maybe remove "on"?

**Comment by Sjors** [comment]
Here's the after screenshot:
rbf

I agree that "Replace-By-Fee" is not clear enough. "Allow increasing transaction fee later" (without "on") is more clear. The footnote mentions the technical terms, which helps users google it.

Although there's plenty of space in the UI now, an short alternative check-box text is: "Allow Fee Increase",

Alternative footnote: 'If the transaction takes a long time to confirm, this allows you to increase the fee later ("Replace-By-Fee", BIP 125). This uses a lower fee by default.'

@wodry tip for your next PR: use a different branch than `master`. E.g. `git checkout -b my-cool-feature`.

**Comment by wodry** [comment]
- Thanks @Sjors, for the screenshot and Your input.
- I am fine with Your and also @laanwj's suggestion to remove the _"on"_ in the button text, and I am fine with the footnote suggestion, too, only _"This uses a lower fee by default."_ I do not understand, so it should be clearer what You mean by that.
- Since I feel unable to cope with the git handling, I would be happy if You or someone else, who is able to do flexible and clean rebasing,  would open a new PR for the relating issue #11344 as a replacement for this PR.

Best Regards

**Comment by flack** [comment]
@Sjors "This uses a lower fee by default." does it really though? I mean at least in the case of custom fee, I would expect it to use the fee you entered. Or do you mean "it allows you to use a lower fee and then increase it later on"? Otherwise I like your alternative text better than the current version

@wodry I can make a new PR if it simplifies handling (but I think you should be fine, using `master` is only inconvenient if you plan on making other changes while this one is still in flight)

**Comment by Sjors** [comment]
@flack see [v0.15 release notes](https://bitcoincore.org/en/2017/09/01/release-0.15.0/):

> Lower fee estimates for RBF users: previously it was difficult to change the fee of unconfirmed transactions after broadcasting them, so Bitcoin Core suggested fees higher than normally needed. As described later in this post, Bitcoin Core now provides tools for increasing the fee of already-sent unconfirmed transactions, so we give lower fee estimates to users of those tools since they can always increase their fee later if necessary.

@wodry:
* this could be a good opportunity to learn more about Git!
* if "This uses a lower fee by default." is confusing to you, then we should find a better phrase...

More generally, it would be nice to have some sort of way to test different wordings, maybe put some A/B testing in the client :-)

**Comment by laanwj** [comment]
As @wodry is not going to maintain this anymore, I'm closing the issue, if anyone still wants to change this label feel free to pick it up.

**Comment by Sjors** [comment]
I'll make a PR.

**Comment by Sjors** [comment]
I made a fresh PR #11556, wording based on my own suggestion [above](https://github.com/bitcoin/bitcoin/pull/11428#issuecomment-338183614).

**Pull request closed without merging**